go.mod

```diff
@@ -195,6 +195,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/simpl
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusexecreceiver => ./receiver/prometheusexecreceiver
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/podmanreceiver => ./receiver/podmanreceiver
```

Comment:
this is not needed yet correct?

Reply:
doesn't harm but not required. removed it.